manifests/v1beta1/installs/katib-with-kubeflow-cert-manager/kustomization.yaml

```diff
@@ -29,7 +29,6 @@ resources:
   # Cert-manager certificate for webhooks
   - certificate.yaml
   # Kubeflow Katib components.
-  - ../katib-with-kubeflow/katib-application.yaml
```

Comment:
Should we also delete also this label: https://github.com/kubeflow/katib/blob/c789f9cf765cd2c729bf3a353a1790097f730f2c/manifests/v1beta1/installs/katib-with-kubeflow-cert-manager/kustomization.yaml#L60-L61 ?